Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/Controls/InfiniteCanvasVirtualDrawingSurface.Render.cs

```diff
@@ -23,7 +23,7 @@ public partial class InfiniteCanvasVirtualDrawingSurface
         private readonly List<IDrawable> _visibleList = new List<IDrawable>();
         private readonly List<IDrawable> _drawableList = new List<IDrawable>();
 
-        internal void ReDraw(Rect viewPort)
+        internal void ReDraw(Rect viewPort, float zoom)
```

Comment:
Rather than passing this in everywhere, we should just internalize this value as a private value and then recalculate it when the `ZoomFactor` property changes or the canvas is resized?

Reply:
I decided for this way as the zoom and view port are linked to each other. Both come from the scrollviewer and can change at the same time based on the user interaction. That's why it should be considered as a single entity or value pair. It is not related to the screen DPI or canvas size.